src/core/configuration/DefaultConfig.ts

```
@@ -166,7 +173,31 @@ export abstract class DefaultServerConfig implements ServerConfig {
+    const maxPlayerForMap = this.maxPlayersForMap(map);
+
+    switch (mode) {
+      case GameMode.FFA:
+        return {
+          maxPlayers: Math.min(150, maxPlayerForMap),
+          numPlayerTeams: undefined,
+        };
+
+      case GameMode.Team:
+        // between 2 and 6 teams
+        const numPlayerTeams = 2 + Math.floor(Math.random() * 5);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)